PIL/Image.py

```diff
@@ -1146,6 +1212,7 @@ def getbbox(self):
         return self.im.getbbox()
 
     def getcolors(self, maxcolors=256):
+        # type: (int) -> Optional[List[Tuple[int, int]]]
```

Comment:
Right, I have to think about this too, with function parameters, as `Optional` can be a misleading term!